docs/source/dev/styleguide.rst

```diff
@@ -30,15 +30,19 @@ find include/ share/picongpu/ share/pmacc -iname "*.def" \
 | xargs clang-format-11 -i
 ```
 
-Format Git Staged Changes
-^^^^^^^^^^^^^^^^^^^^^^^^^
+Format Changes With Git
+^^^^^^^^^^^^^^^^^^^^^^^
 
-To format all changed added with `git add` you can run `git clang-format-11`.
+Instead of using the bash command above you can use *Git* together with *clang-format* to format your patched code, only.
+You must extent your local git configuration **once** with all file endings used in PIConGPU:
```

Comment:
Instead of using the bash command above you can use *git* together with clang-format to format your patched code only. Before applying this command, you must extend your local git configuration **once** with all file endings used in PIConGPU: